datahub-web-react/src/app/useAppConfig.ts

```diff
@@ -12,3 +12,8 @@ export function useIsShowAcrylInfoEnabled() {
     const appConfig = useAppConfig();
     return appConfig.config.featureFlags.showAcrylInfo;
 }
+
+export function useIsShowAccessManagementEnabled(): boolean {
```

Comment:
Once we pass the appConfig into the visible function, we should no longer need this method

Reply:
removed useIsShowAccessManagementEnabled()